JobRecommendationService.md

```diff
@@ -0,0 +1,173 @@
+# Job Recommendation service
+# Contents
+
+- [Summary](#summary)
+- [Request structure](#request-structure)
+  - [JobData](#jobdata)
+    - [Job](#job)
+  - [UserData](#userdata)
+  - [UserPreferences](#userpreferences)
+    - [PreferredLocation](#preferredlocation)
+    - [Compensation](#compensation)
+    - [CaroteneTitle](#carotenetitle)
+    - [CaroteneVersion](#caroteneversion)
+- [Response Structure](#response-structure)
+- [Versioning](#versioning)
+
+----------------------------
+# Summary
+
+Job Recommendation Service is part of the profile recommendations, which recommends similar and related jobs to a user based on the jobs he applied and user preferences. The recommendations are combined from FAISS indexes and GBR data, then filtered by the applied jobs.
+
+The endpoint for Job Recommendation Service is at https://api.careerbuilder.com/core/recommendations/job
+
+----------------------------
+# Request structure
+The service supports GET and POST requests to:
+https://api.careerbuilder.com/core/search/candidatereranker.
```

Comment:
I think this is left over copy paste?

Reply:
Also, if you do fix this then it seems to make line 22 superfluous.

Reply:
you got me. Thanks.